health-endpoint-monitoring/HealthEndPointMonitoring.Web/Web.config

```diff
@@ -80,6 +82,18 @@
         <assemblyIdentity name="Microsoft.Data.Edm" publicKeyToken="31bf3856ad364e35" culture="neutral" />
         <bindingRedirect oldVersion="0.0.0.0-5.7.0.0" newVersion="5.7.0.0" />
       </dependentAssembly>
+      <dependentAssembly>
```

Comment:
Does Web.config need reference to the new SDKs?

Reply:
Not necessary , adding reference to the new SDKs in csproj is enough.